examples/plot_anomaly_comparison.py

```diff
@@ -0,0 +1,120 @@
+"""
+========================================================================
+Comparing different anomaly/novelty detection algorithms on toy datasets
+========================================================================
+
+This example shows characteristics of different
+anomaly/novelty detection algorithms on 2D datasets. Datasets contain
+one or two modes (regions of high density) to illustrate the ability
+of algorithms to cope with multimodal data.
+
+While these examples give some intuition about the
+algorithms, this intuition might not apply to very high
```

Comment:
Is it possible to expand on this or provide a reference? What features of the shown decision functions would not transfer to higher dimensions?

Reply:
Well it's the same sentence as for our clustering summary example.